site/blog/2025-09-12-announcing-teaclave-trustzone-sdk-0.6.0.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Announcing Apache Teaclave™ TrustZone SDK 0.6.0
+date: 2025-09-12
+author: Yuan Zhuang
+---
+
+On behalf of the Teaclave community, I am happy to announce the release of
+Teaclave TrustZone SDK 0.6.0.
+
+Teaclave empowers developers to build memory-safe Trusted Applications across
+diverse confidential computing platforms, including Intel SGX and Arm TrustZone.
+Apache Teaclave TrustZone SDK, a subproject of Apache Teaclave, is based on the
+OP-TEE project, a widely used open-source TrustZone OS.
+
+This release aligns with the OP-TEE quarterly release schedule and is compatible
+with OP-TEE 4.7.0. It introduces major improvements across features, build
+workflows, and documentation. Highlights include new TLS client and server
+examples built on the latest Rustls, a significant refactoring of the Object
+component for better maintainability, and enhancements to the build system and
+CI pipeline with separated test workflows, new environment configurations for
+building standard TAs, and simplified Makefiles. Documentation has also been
+refreshed for consistency, clarity, and an improved developer experience.
+
+For more details, please refer to our release notes:
+https://github.com/apache/teaclave-trustzone-sdk/releases/tag/v0.6.0
+
+A link to the download can be found here:
```

Comment:
These two lines (download link to dist.apache.org) are more appropriate for the vote stage and don’t need to appear in the blog post. Since the blog already includes a dedicated Download section pointing to the correct download page, that’s sufficient for readers.